core/git/git.test.ts

```diff
@@ -165,6 +165,48 @@ Deno.test("git().clone({ singleBranch }) copies a single branch", async () => {
   );
 });
 
+Deno.test("git().clone() without directory creates repo directory", async () => {
```

Comment:
Make this test similar to other tests around here.  Notice the differences in style.

The test order should be higher, since this is the default behavior. There should be a test for the new `directory` option, right below the core `clone()` tests that do not test with options.